### ISSUE TITLE
Make it detect steghide success

### ISSUE THREAD

**Issue by bee-san** [StegKraken/StegKraken#3]
When we run `steghide` it either returns a 0 or a 1 for success.

If it returns 0, we want to return from the program and print the password as we were successful.

This is the documentation on it:
https://doc.rust-lang.org/std/process/struct.ExitStatus.html

This is the function to add the code to.
https://github.com/StegKraken/StegKracken/blob/master/src/main.rs#L64

**Algorithm**
1. Store the result of the command in a variable.
2. If command is successful:
    * Print password
    * Print "we are successful" or something
3. Else it's not successful, just carry on - don't do anything. You probably don't need to write code for this stuff, just no else statement.

**Comment by s1l3nt78** [comment]
Did you create this in order to keep track of development,
or for someone to implement it? 🙂

**Comment by bee-san** [comment]
ah hey! 

this is still heavily in development, we just have a minimal viable product out right now :))

not sure when we expect to release but likely this month :)

**Comment by s1l3nt78** [comment]
No problem, was just going to say I'd help implement it if needed but no worries.

Though happy to help with anything if ever needed 🙂

**Comment by bee-san** [comment]
> No problem, was just going to say I'd help implement it if needed but no worries.
> 
> Though happy to help with anything if ever needed slightly_smiling_face

Are you a C++ programmer by any chance? We're switching from Rust to C++ :)

**Comment by s1l3nt78** [comment]
Yeah I can help with the migration 🙂

**Comment by s1l3nt78** [comment]
If your okay with it, I'll start with it and put it into a private repo and give you access to to edits as you please.

**Comment by bee-san** [comment]
Hey!

Do you mind waiting? The only thing we have left is to import Steghide, which is being worked on by someone else right now (they're finishing some work on the 10th so will be more active then) (and also, none of us have coded C++ before so our code will be a mess haha).

Can you join the Discord? :) 

https://discord.gg/gfwzV9U

**Comment by s1l3nt78** [comment]
Absolutely! I wont go forward with anything until I get the go ahead, and no problem. I work best with chaos 🙂 

Will join up to the server this eve and just start to take a look at the code to see about how to properly do the migration
